Day12/src/index.ts

```diff
@@ -0,0 +1,46 @@
+const Koa = require('koa');
+const app = new Koa();
+const json = require('koa-json');
+const koaroute = require('@koa/router');
+const bodyparser = require('koa-bodyparser');
+const port = 3000;
+const router = new koaroute();
+const studntController = require('./controllers/studentcontroller');
+const resultController = require('./controllers/resultContoller');
+app.use(json());
+app.use(bodyparser());
+
+router.get('/', (ctx: any) => {
+    ctx.body = "hello";
+})
+
+router.get('/student',studntController.getStudents);
```

Comment:
we can use combineRoute function. no need to change this just wanted to mention.